src/component/common/InstanceStatus/InstanceStatusBar.tsx

```diff
@@ -72,6 +72,7 @@ const calculateTrialDaysRemaining = (
         : undefined;
 };
 
+// TODO - Cleanup to use theme instead of colors
```

Comment:
These components were using colors directly, which we should avoid. Added a comment for us to clean up later.